webpages/js/userprofile.js

```diff
@@ -48,23 +48,32 @@
         window.location.reload();
       }
     });
+    _.addEventListener('#personalinfo .register', 'click', function() {
+      window.location.href = '/register';
+    });
   }
 
   function fillUserProfile(user) {
-    lima.userPageIsAbout = user;
+    if (JSON.stringify(user) != JSON.stringify({})) { // we have a user
```

Comment:
this block is probably not needed anymore, because we shouldn't be calling /api/profile first